### PR TITLE
Add jest instructions to readme

### PR DESCRIPTION
**Pull request by keyserfaty**
Hey :) Loved your project, it was a lifesaver.

My tests were breaking because of the library so I thought I might add what I did to your readme.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -256,3 +256,20 @@ const config = {
 }
 
 ```
+
+### How to use with Jest?
+
+**A**: Add a `react-native-render-html-table-bridge.js` file to your `__mocks__` folder (this folder should be at the root of your project) with the following content:
+
+```
```
Comment (jsamr) [inline readme.md]: Could you just add js tag after the ``` to have syntax highlight?

**Comment by jsamr** [comment]
@keyserfaty Very nice of you. See my comment.

File: readme.md (hunk 1 above)
Comment (keyserfaty) [inline readme.md]:
```suggestion
``` js
```

**Comment by keyserfaty** [comment]
Done.

**Pull request merged**